modules/ROOT/pages/osv2-apis.adoc

```diff
@@ -108,12 +110,32 @@ fills in this information for you.
 Set up an application like Postman to access the object store.
 Obtain the `access_token` value:
 
-. Configure the object store access URL to: +
-`{{osBaseUrl}}/api/v1/organizations/:ORGANIZATION_ID/environments/:environmentId/stores/myTestStoreId/objects`
+. Configure the object store access URL to:
++
+`_osBaseUrl_/api/v1/organizations/:ORGANIZATION_ID/environments/:environmentId/stores/myTestStoreId/objects`
```

Comment:
is it easy to tell reader where to obtain the value for osBaseUrl (this question may be out of scope).